conf/locale/locale_en-US.ini

```diff
@@ -485,8 +485,7 @@ issues.commit_ref_at = `referenced this issue from a commit <a id="%[1]s" href="
 issues.poster = Poster
 issues.collaborator = Collaborator
 issues.owner = Owner
-issues.sign_up_for_free = Sign up for free
-issues.sign_in_require_desc = to join this conversation. Already have an account? <a href="%s">Sign in to comment</a>
+issues.sign_in_require_desc = <a href="%s">Sign in to comment</a>
```

Comment:
In this case, I think we should put `<a></a>` part directly in the template, and only keep translate words in locale file.